minio.json

```diff
@@ -15,10 +15,15 @@
                 "launch_order": 1,
                 "ports": {
                     "9000": {
-                        "description": "API and Portal access, default 9000",
+                        "description": "Port for API access, default 9000",
                         "host_default": 9000,
-                        "label": "MinIO API and Portal",
-                        "protocol": "tcp",
+                        "label": "MinIO API Port",
+                        "ui": true
```

Comment:
@MikeMcPherson ,

Is the `ui: true` correct here? Port 9001 is already having this tag so a duplicate `"ui"` might throw Rockstor off and create inconsistency in the webUI link that Rockstor generates. This might also explain the discrepancy between what you see and what @phillxnet sees.